packages/components/pie-modal/src/index.ts

```diff
@@ -233,7 +240,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
      * @private
      */
     private renderLeadingAction (): TemplateResult | typeof nothing {
```

Comment:
i might be missing something here, but don't we want to allow users to trigger an on click callballback for the action buttons here? they both close the modal at the moment.

Reply:
Not sure if there is a ticket for this but the markup that was there before both closed the dialog, I'm guessing this is just intentional until we have a tickets for this work. @xander-marjoram @fernandofranca do you know if we have one? If not i can add the changes here.